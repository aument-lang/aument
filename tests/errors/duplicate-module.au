import "" as a;
import "" as a;