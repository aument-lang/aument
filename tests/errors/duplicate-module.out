parser error(9) in -: this module is already imported
2 | import "" as a;
                 ^
